caiman/motion_correction.py

```diff
@@ -2557,8 +2558,8 @@ def motion_correction_piecewise(fname, splits, strides, overlaps, add_to_movie=0
     if save_movie:
         if base_name is None:
             base_name = os.path.split(fname)[1][:-4]
-        fname_tot = base_name + '_d1_' + str(dims[0]) + '_d2_' + str(dims[1]) + '_d3_' + str(
-            1 if len(dims) == 2 else dims[2]) + '_order_' + str(order) + '_frames_' + str(T) + '_.mmap'
+        fname_tot = base_name + '_d1_' + str(dims[0]) + '_d2_' + str(dims[1]) + '_d3_1' + \
```

Comment:
Stared carefully at the preceding code and convinced myself that dims never has a third dimension here.

Longer term we may want to make utility functions that handle constructing names for memmap files (possibly with a default path).

Reply:
@pgunn This might change in the future if we enable 3d motion correction. Can you revert?